ext/ffi/00_ffi.js

```diff
@@ -138,6 +151,13 @@
       }
       return core.opSync("op_ffi_ptr_of", value);
     }
+
+    static getPointerValue(pointer) {
+      if (typeof pointer !== "object") {
+        throw new TypeError("Invalid FFI pointer value");
+      }
```

Comment:
This error seems duplicated as `op_ffi_get_ptr_value` already does the same?

Reply:
Yup, not really needed.